apps/dashboard/src/components/icons/circle-check.tsx

```diff
@@ -1,11 +1,11 @@
 import type { HTMLAttributes } from 'react';
 
-export const CircleCheck = (props: HTMLAttributes<HTMLOrSVGElement>) => {
+export const CircleCheck = (props: HTMLAttributes<HTMLOrSVGElement> & { color?: string }) => {
   return (
     <svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 17 16" fill="none" {...props}>
       <path
         d="M6.50004 7.99999L7.83337 9.33333L10.5 6.66666M15.1667 7.99999C15.1667 11.6819 12.1819 14.6667 8.50004 14.6667C4.81814 14.6667 1.83337 11.6819 1.83337 7.99999C1.83337 4.3181 4.81814 1.33333 8.50004 1.33333C12.1819 1.33333 15.1667 4.3181 15.1667 7.99999Z"
-        stroke="#1FC16B"
+        stroke={props.color ?? '#1FC16B'}
```

Comment:
Added the abilty to update the color from the default green

Reply:
Variant styles should be applied via Tailwind classes on the `className` prop only, not via new props. So you can simply pass something like: `<CircleCheck className="stroke-accent" />`, or replace `accent` with an override using `stroke-[red]` or whichever color is necessary.

Reply:
@rifont issue is that I need this to be applied to the specific path element which is a child of the SVG parent. Will check if I can somehow convert it to use nesting.